apps/api/src/hyp3_api/handlers.py

```diff
@@ -34,39 +31,21 @@ def is_uuid(val):
     return True
 
 
-@connexion_app.app.before_request
-def check_system_available():
-    if environ['SYSTEM_AVAILABLE'] != "true":
-        message = 'HyP3 is currently unavailable. Please try again later.'
-        error = {
-            'detail': message,
-            'status': 503,
-            'title': 'Service Unavailable',
-            'type': 'about:blank'
-        }
-        return make_response(jsonify(error), 503)
-
-
-@connexion_app.app.route('/')
-def redirect_to_ui():
-    return redirect('/ui')
-
-
 def post_jobs(body, user):
     print(body)
 
     monthly_quota = get_max_jobs_per_month(user)
     remaining_jobs = util.get_remaining_jobs_for_user(user, monthly_quota)
     if remaining_jobs - len(body['jobs']) < 0:
         message = f'Your monthly quota is {monthly_quota} jobs. You have {remaining_jobs} jobs remaining.'
-        return problem(400, 'Bad Request', message)
+        abort(error(400, 'Bad Request', message))
```

Comment:
These could just be one call to error, and that function could call abort, but I like that abort is here since it is more obvious that this will exit